presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDirectSelectiveStreamReader.java

```diff
@@ -293,7 +298,7 @@ private int readWithFilter(int[] positions, int positionCount, int dataLength)
                     }
                     else {
                         assert length == 0;
-                        if (filter.testBytes("".getBytes(), 0, 0)) {
+                        if (rowGroupFilter.testBytes("".getBytes(), 0, 0)) {
```

Comment:
While we are on it: can you please replace `"".getBytes()` with a static final `new byte[0]`? Or double-check what is going on here.

Reply:
Will address in a follow up change.